client/tests/unit/applicant-info-1.spec.ts

```diff
@@ -5,7 +5,9 @@ import Vuetify from 'vuetify'
 import $intJurisdictions from '@/store/list-data/intl-jurisdictions'
 import $canJurisdictions from '@/store/list-data/canada-jurisdictions'
 
-const store = newReqModule
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
```

Comment:
bueno!

Reply:
Yes but needs to be used sparingly as it seems to break some tests (and, in some cases, it's just not needed).